source/_patterns/10-atoms/buttons/load-more.scss

```diff
@@ -5,6 +5,9 @@
   border-width: 1px 0;
   cursor: pointer;
   width: 100%;
+  display: flex;
```

Comment:
hast Du's mal im Safari angekuckt? (der hat meines Wissens gewisse Mühe mit flex-Buttons)

Reply:
Sieht ganz gut aus in Safari (z.B. http://www-test.srf.ch/sport/tennis/atp-tour/sieg-gegen-nishikori-in-madrid-djokovic-mit-einem-ausrufezeichen)

Reply:
👍  ausser: <img width="525" alt="bildschirmfoto 2018-07-09 um 14 30 26" src="https://user-images.githubusercontent.com/7767385/42450551-cf21f072-8384-11e8-97eb-b6f4369715a8.png"> (im Safari - bei Kommentare «Mehr anzeigen» im verlinkten Artikel)

Reply:
wie besprochen - das passiert beim Safari in Sierra - in El Capitan war's noch in Ordnung. Ich schau grad...

Reply:
Ist gefixt. Hab's ganz frech in ein div umgewandelt. Ansonsten müssen wir einen rechten Handstand nur für Safari auf Sierra+ machen...